articles/openshift/oshift-how-view-proxy-logs.md

```diff
@@ -0,0 +1,57 @@
+---
+title: How to view proxy logs in a multi-network OpenShift cluster | UKCloud Ltd
+description: Explains how to view the logs of the squid proxy in your cluster
+services: openshift
+author: Kieran O'Neill
+reviewer:
+lastreviewed: 03/12/2019
+
+toc_rootlink: How To
+toc_sub1:
+toc_sub2:
+toc_sub3:
+toc_sub4:
+toc_title: View proxy logs in a multi-network OpenShift cluster
+toc_fullpath: How To/oshift-how-view-proxy-logs.md
+toc_mdlink: oshift-how-view-proxy-logs.md
+---
+
+# How to view proxy logs in a multi-network OpenShift cluster
+
+## Overview
+
+UKCloud for OpenShift enables you to develop, deploy, and manage digital and container-based applications seamlessly across local physical or virtual environments, with full portability to and from UKCloud.
+
+This article explains how you can view the connection logs generated by the squid proxy in your OpenShift cluster to identify potential connection issues, such as attempting to access endpoints that aren't whitelisted.
+
+### Prerequisites
+
+Your OpenShift cluster must have access to multiple networks, be version 3.11 or later, and must have proxy logging enabled. Any OpenShift clusters deployed from November 2019 will have proxy logging enabled. If you have an older OpenShift cluster and would like proxy logging enabled, raise a request on the UKCloud Portal for this functionality.
+
+## Viewing logs
+
+In multi-network 3.11 clusters, we run a squid proxy on the control plane load balancers to access Red Hat's container registry `registry.redhat.io`. The proxy also allows you to hit internet endpoints from community network nodes, providing you pass the correct variables to your pods and have whitelisted the endpoint. You may inadvertently be traversing the proxy where you do not want to or attempting to hit endpoints that are not whitelisted, which will cause you to get 403 forbidden errors. The proxy logs contain information about these errors.
+
+As a customer, while you don't have access to the logs on the control plane load balancers, we send them through to a set of fluentd pods in a project called `squid-showback`. These logs are then forwarded on to the clusters' Aggregated Logging and you can view them in the Kibana dashboard in the `openshift-logging` project.
+
+Set a filter of `kubernetes_namespace_name: "squid-showback"` to filter only for the logs from the pods receiving the proxy logs and show only the message. You can then query the message field for the particular IP/endpoint you're looking for. From here you can see if it's being denied and update the whitelist as needed.
```

Comment:
```suggestion
Set a filter of `kubernetes_namespace_name: "proxy-showback"` to filter only for the logs from the pods receiving the proxy logs and show only the message. You can then query the message field for the particular IP/endpoint you're looking for. From here you can see if it's being denied and update the whitelist as needed.
```

Can this also be updated to proxy instead of squid please?